rollbar-java/src/main/java/com/rollbar/notifier/Rollbar.java

```diff
@@ -693,4 +699,103 @@ private void sendPayload(Config config, Payload payload) {
       config.sender().send(payload);
     }
   }
+
+  /**
+   * Construct a notifier defaults for everything including Sender.
+   * Caution: default sender is slow and blocking. Consider providing a Sender overload.
+   * @param accessToken not nullable, the access token to send payloads to
+   * @param environment not nullable, the environment to send payloads under
+   */
+  @Deprecated
+  public Rollbar(String accessToken, String environment) {
```

Comment:
We are adding the constructors from the previous version but the packaging has changed so it will imply to update the user's code to use the new packaging. 
We could add a class with the same previous API at `com.rollbar` level and implement the old API in it, using delegation to this class. We could also mark as `depecreated` this whole class.